coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -341,6 +351,11 @@ def archify(f):
             logger.warning('Continuing...')
             return
 
+        self.tag_rpms(desiredrpms)
+
+    @catch_exceptions_and_continue
```

Comment:
I don't think we need the `@catch_exceptions_and_continue` at the top of all of these functions do we? In the original file there were only two.. one for the "initial run" path and one for the "processing messages" path. If a function is called from another function that has `@catch_exceptions_and_continue` then it will be caught at the top level.

Reply:
Yes, absolutely. I was incorrectly cargo culting there.